runtime/master/src/main/java/org/apache/nemo/runtime/master/resource/ResourceSpecification.java

```diff
@@ -29,22 +29,34 @@
   private final String containerType;
   private final int capacity;
   private final int memory;
+  private final double maxOffheapRatio;
+  private final int maxOffheapMb;
   private final int poisonSec; // -1 if this resources is not poisoned
 
   public ResourceSpecification(final String containerType,
                                final int capacity,
                                final int memory) {
-    this(containerType, capacity, memory, -1);
+    this(containerType, capacity, memory, 0.02, -1);
   }
 
   public ResourceSpecification(final String containerType,
```

Comment:
I think this constructor wouldn't be needed, as the constructor above is only available for testing. Let's remove this unused constructor.

Reply:
I removed the constructor. Thanks for your comment.